Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -54,46 +54,51 @@ def self.capture_output(&block)
       output
     end
 
+    def fetch
+      odebug "Hbc::Installer#fetch"
+
+      satisfy_dependencies
+      verify_has_sha if @require_sha && !@force
+      download
+      verify
+    end
+
+    def stage
+      odebug "Hbc::Installer#stage"
+
+      extract_primary_container
+      save_caskfile
+    rescue StandardError => e
+      purge_versioned_files
+      raise e
+    end
+
     def install
-      odebug "Hbc::Installer.install"
+      odebug "Hbc::Installer#install"
 
       if @cask.installed? && !force
         raise CaskAlreadyInstalledAutoUpdatesError, @cask if @cask.auto_updates
         raise CaskAlreadyInstalledError, @cask
       end
 
       print_caveats
-
-      begin
-        satisfy_dependencies
-        verify_has_sha if @require_sha && !@force
-        download
-        verify
-        extract_primary_container
-        install_artifacts
-        save_caskfile
-        enable_accessibility_access
-      rescue StandardError => e
-        purge_versioned_files
-        raise e
-      end
+      fetch
+      stage
+      install_artifacts
+      enable_accessibility_access
 
       puts summary
     end
 
     def summary
-      s = if MacOS.version >= :lion && !ENV["HOMEBREW_NO_EMOJI"]
-        (ENV["HOMEBREW_INSTALL_BADGE"] || "\xf0\x9f\x8d\xba") + "  "
-      else
-        Formatter.headline("Success! ", color: :blue)
```

Comment:
Yes, because the headline seems redundant when we already have "successfully" in the message.